### PR TITLE
Feature/cicd deployment improvements

### PR DESCRIPTION
**Pull request by cschreep**
This PR introduces improvements surrounding build and deployment processes

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -15,5 +15,16 @@ pipeline {
...
             }
         }
+        stage('Publish') {
+            when {
+                branch 'develop'
+            }
+            steps {
+                sh '''
+                make build.image
+                make publish.image
```
Comment (andrew-rti) [inline Jenkinsfile:25]: so simple!

**Review by andrew-rti — DISMISSED** [review]
LGTM!

**Review by alexwaldrop — COMMENTED** [review]
Love the devops changes. My only concern with the a single 'dug'-level logger is that it's often been helpful to track down bugs by using the loggers to help point to where we were in program execution. That's kind of janky though. I'm honestly just interested in understanding the motivation for the single logging level.

**Review by alexwaldrop — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -21,8 +21,6 @@ USER $USER
 WORKDIR $HOME
 
 ENV PATH=$HOME/.local/bin:$PATH
-ENV ELASTIC_API_HOST=
-ENV ELASTIC_API_PORT=
```
Comment (andrew-rti) [inline Dockerfile:25]: 💯

**Review by andrew-rti — APPROVED** [review]

**Pull request merged**